src/js/PluginMountPoints.js

```diff
@@ -0,0 +1,2 @@
+var {PluginMountPoints} = window["marathonPluginInterface"];
```

Comment:
Yeah. But these files aren't wrapped by React. BTW: I couldn't think of a good reason why we shouldn't explicitly read this from the window object as it's the way the whole plugin loading mechanism works.